### PR TITLE
Mobile First Pesign Page

### PR DESCRIPTION
**Pull request by Ejikeokegbe**
Added linter for error handling
The installing of webhint on the project is for easy access
Added stylelintrc for good styleling and style error fixing
Design the page with Figma template and flexbox  
Added README file for content and information

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (joel-kalema) [inline index.html:3]: - [ ] You did great work 😃, however, you need to fix some parts. For good syntax, Kindly make sure you follow this structure:

`````
<body>
    <main>
        <header>
               <a href="#">LOGO</a>
               <nav>
                    --- hamburger menu ---
               </nav>
        </header>
        <section>
               --- the headline section (right after the header) ----
        </section>
    </main>
</body>
`````


- [ ]  Kindly make sure your website should look exactly like the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/proto/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-122&scaling=min-zoom&page-id=0%3A1) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=Your%20website%20should%20look%20exactly%20like%20the%20design%20(e.g.%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.) 😃,

> Your `background image`
> Your h2 : `color: #172B4D;`
> Your p: `color: #344563;` 


> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |Screenshot 2023-05-04 122601| |Screenshot 2023-05-04 122734| |

**Review by joel-kalema — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @Ejikeokegbe 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @joel-kalema  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+}
+
+section p {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  margin-right: 70px;
```
Comment (tufoinnkuo10) [inline style.css:94]: - [ ] please remove the margin right so the text can be centered.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+  font-size: 18px;
+  line-height: 20px;
+  margin: auto 0;
+}
+
+#header {
+  background-image: url("pimages/header-background-mobile.png");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: contain;
```
Comment (tufoinnkuo10) [inline style.css]: - [ ] kindly use background size ..` cover` here, so it covers the whole page.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio Page</title>
+</head>
+<body>
+  <main>
+    <header>
+      <nav class="flexr container">
+        <a href="#" class="logo">
```
Comment (tufoinnkuo10) [inline index.html:14]: - [ ] as demonstrated by the previous reviewer, the logo is not inside the nav tag. please kindly follow that for semantic reasons.
- [ ] please centralize your headline heading and text so it matches the design. 
- [ ] kindly add some padding between the lets connect, the text above it and the social links below it. 
yours
Screenshot (1056)_LI
original
Screenshot (1057)

File: README.md [hunk 5]
```diff
@@ -1 +1,253 @@
...
+
+  - [Answer_2]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (tufoinnkuo10) [inline README.md:251]: - [optional] please, your MIT link is broken. Kindly set it up .we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @Ejikeokegbe 

nice work so far. There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ Figma design well followed

Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @Ejikeokegbe,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**